create-release.py

```diff
@@ -77,7 +77,12 @@ def dependency_exists(command) -> bool:
 def sed(file: str, pattern: str, replace: str) -> None:
     """Perform regex substitution on a given file"""
     try:
-        check_run(["sed", "-i", "", "-e", f"s#{pattern}#{replace}#g", file], capture_output=False)
+        if sys.platform == "linux" or "linux2":
```

Comment:
This will always be true.  I would suggest a list since its more extensible...
```suggestion
        if sys.platform in ["linux", "linux2"]:
```

Reply:
woops, gracias for the review =)